BfA/Underrot/Trash.lua

```diff
@@ -326,6 +328,24 @@ do
 	end
 end
 
+
+function mod:WickedEmbraceApplied(args)
+	if self:Me(args.destGUID) or self:Healer() or self:Dispeller("magic") then
+		local healthPercent = (UnitHealth(args.destName) / UnitHealthMax(args.destName)) * 100
```

Comment:
There is a self:GetHealth() that provides this calculation, but if the ability is so dangerous then it seems like playing alert all the time would be fine. With how spiky health can be in some of these pulls I worry that the two different sounds might just be confusing.

Reply:
Makes sense since it is relative to key level anyways as far as "dangerous" health goes.